### PR TITLE
Fixes sight range on lootpanel [no gbp]

### LINKED ISSUE #82768: Alt-clicking things in your inventory or storage does not work.
## Issue Summary

Alt-clicking objects in your inventory or storage does not work. Period.

## Testmerges:

[#82298: 'Wawastation' by mc-oofert at commit e229259992](https://www.github.com/tgstation/tgstation/pull/82298)
[#82765: 'Fixes sight range on lootpanel [no gbp]' by jlsnow301 at commit f225ebf277](https://www.github.com/tgstation/tgstation/pull/82765)

## Reproduction:

Equip something with an alt-click functionality into an equippable slot.

Observe that nothing happens when you attempt to use that functionality.

### LINKED ISSUE #82772: Ai cant use the new Alt click Item menu
## Issue Summary
Prior to the change to the lovely new rpg menu, AI players could alt click tiles to see everything on them. Ai players do not have the ability to use the new rpg alt menu

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
When asked to make lootpanel visible at range they did not just mean _any range_
AI alt click was a little weird (still is) but now it works

(TM BUG)
Fixes #82768
## Why It's Good For The Game
Bug/exploit fix
Fixes #82772
## Changelog
:cl:
fix: Fixes the sight range on lootpanel.
/:cl:

### CODE REVIEW

File: code/_onclick/click_alt.dm [hunk 1]
```diff
@@ -6,32 +6,32 @@
 /mob/proc/base_click_alt(atom/target)
 	SHOULD_NOT_OVERRIDE(TRUE)
 
-	var/turf/tile = isturf(target) ? target : get_turf(target)
+	// Is it visible (and we're not wearing it (our clothes are invisible))?
+	if(!(src in viewers(7, target)) && !Adjacent(target))
```
Comment (Kapu1178) [inline code/_onclick/click_alt.dm:10]:
```suggestion
	if(!(src in viewers(7, target)) && !CanReach(target))
```

File: code/_onclick/click_alt.dm [hunk 2]
```diff
@@ -10,32 +10,31 @@
 	if(SEND_SIGNAL(src, COMSIG_MOB_ALTCLICKON, target) & COMSIG_MOB_CANCEL_CLICKON)
 		return
 
+	// Is it visible (and we're not wearing it (our clothes are invisible))?
+	if(!(src in viewers(7, target)) && !CanReach(target))
```
Comment (ZephyrTFA) [inline code/_onclick/click_alt.dm:14]:
```suggestion
	if(!(src in viewers(7, target)) && !CanReach(target))
		return
```

File: code/_onclick/ai.dm [hunk 3]
```diff
@@ -121,6 +121,26 @@
...
+		if(SEND_SIGNAL(target, COMSIG_CLICK_ALT, src) & CLICK_ACTION_ANY)
+			return
+
+		// AI alt click interaction succeeds
+		if(target.ai_click_alt(src) & CLICK_ACTION_ANY)
+			return
+
+	var/turf/tile = get_turf(target)
+
+	client.loot_panel.open(tile)
```
Comment (MrMelbert) [inline code/_onclick/ai.dm:141]: Have you tested this with AI static? 

Like can you open the loot panel over tiles you don't have camera view of?

Comment (MrMelbert) [inline code/_onclick/ai.dm:141]:
```suggestion
	client.loot_panel.open(get_turf(target))
```

Comment (jlsnow301) [reply code/_onclick/ai.dm:141]: yes i have

**Pull request merged**